### ISSUE TITLE
button difficult to discern whether to press or not

### ISSUE THREAD

**Issue by lijiachang** [variar/klogg#706]
<img width="260" alt="image" src="https://github.com/variar/klogg/assets/28041718/1db2b9bb-ad57-4e5f-a55f-d489743327ca">

Most obvious under the dark theme

**Comment by lijiachang** [comment]
i dont know who is pressed

**Comment by variar** [comment]
Indeed. If someone could provide better colors for dark themes I can integrate them.
